docs/modules/general/pages/implementation-details/bootstrapping.adoc

```diff
@@ -1,26 +1,32 @@
 = OneCX Microfrontend Content Bootstrapping
 
-:mfe-loading_url: xref:implementation-details/mfe-content-loading/index.adoc
-:angular-webcomponents_url: xref:implementation-details/mfe-content-loading/webcomponents.adoc#onecx/angular-webcomponents
-:angular_issues_url: xref:implementation-details/mfe-content-loading/webcomponents.adoc#angular_issues
-:createAppEntrypoint_url: xref:implementation-details/mfe-content-loading/webcomponents.adoc#create_app_entrypoint
-:startsWith_url: xref:implementation-details/mfe-content-loading/webcomponents.adoc#starts_with
-:bootstrapRemoteComponent_url: xref:implementation-details/mfe-content-loading/webcomponents.adoc#bootstrap_remote_component
-:initializeRouter_url: xref:implementation-details/mfe-content-loading/webcomponents.adoc#initialize_router
-:events_topic_url: xref:latest@guides:javascript:libraries/integration-interface.adoc#events_topic
-:current_mfe_topic_url: xref:latest@guides:javascript:libraries/integration-interface.adoc#current_mfe_topic
-:user_service_url: xref:latest@guides:angular:libraries/angular-integration-interface.adoc#user_service
+:idprefix:
+:idseparator: -
 
+:mfe-loading_url: xref:implementation-details/mfe-content-loading/index.adoc
+:angular-webcomponents_url: xref:implementation-details/mfe-content-loading/webcomponents.adoc#onecx-angular-webcomponents
+:angular_issues_url: xref:implementation-details/mfe-content-loading/webcomponents.adoc#angular-issues
+:createAppEntrypoint_url: xref:implementation-details/mfe-content-loading/webcomponents.adoc#create-app-entrypoint
+:startsWith_url: xref:implementation-details/mfe-content-loading/webcomponents.adoc#starts-with
+:bootstrapRemoteComponent_url: xref:implementation-details/mfe-content-loading/webcomponents.adoc#bootstrap-remote-component
+:initializeRouter_url: xref:implementation-details/mfe-content-loading/webcomponents.adoc#initialize-router
```

Comment:
don't mix separators? or add separators

Reply:
Same as above, for methods I went with the whole method name with casing as for js methods